common/OpTestHMC.py

```diff
@@ -481,6 +481,25 @@ def get_lmb_size(self):
         '''
         return self.run_command("lshwres -r mem -m %s --level sys -F mem_region_size" % self.mg_system)
 
+    def configure_16gb_hugepage(self, num_hugepages):
```

Comment:
why not a generic name ? like configure_hugepage() ?

Reply:
Because we use this command to get only 16gb hugepages data...

Reply:
yup we should have a generic function to configure hugepages. The value will be provided by the user.

Reply:
okay will rename it

Reply:
i feel 16gb is fine as in CEC we able to configure 16GB only if it open it mis guide user